docs/visualization/visualize_unit_responses.ipynb

```diff
@@ -83,7 +83,7 @@
      "name": "stderr",
```

Comment:
<p>This is probably better visualize by an image where channel number is the vertical axis and xaxis is time. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>